pkg/extkingpin/path_content_reloader_test.go

```diff
@@ -57,13 +57,15 @@ func TestPathContentReloader(t *testing.T) {
 					testutil.Ok(t, pathContent.Rewrite([]byte("test modified again")))
 				},
 			},
-			wantReloads: 1,
+			wantReloads: 2,
 		},
 		{
 			name: "Chmod doesn't trigger reload",
 			args: args{
 				runSteps: func(t *testing.T, testFile string, pathContent *staticPathContent) {
 					testutil.Ok(t, os.Chmod(testFile, 0777))
+					testutil.Ok(t, os.Chmod(testFile, 0666))
+					testutil.Ok(t, os.Chmod(testFile, 0777))
```

Comment:
Why does this trigger 2 reloads now?

Reply:
This one triggers zero reload... see line 71 below.

The one above it triggers 2 reloads, which is truly what we expect, because the file is rewritten twice. Previously it could pass due to the code being much faster than the async fs watcher notification for the second rewrite of the file.